cli/commands/env.go

```diff
@@ -0,0 +1,33 @@
+package commands
```

Comment:
I can't quite tell (perhaps simply a limited golang understanding)  -- if these methods are lowercase, they aren't available to the user invoking the cli?